lib/helpers.js

```diff
@@ -1,14 +1,18 @@
 'use strict';
 
+// BUG: quick fix
+Error.captureStackTrace = () => { };
+Error.prototype.captureStackTrace = () => { };
+
 var uri = require('url');
 
-var ValidationError = exports.ValidationError = function ValidationError (message, instance, schema, path, name, argument) {
-  if(Array.isArray(path)){
+var ValidationError = exports.ValidationError = function ValidationError(message, instance, schema, path, name, argument) {
+  if (Array.isArray(path)) {
```

Comment:
Please try to keep changes strictly relevant to the purpose of the PR... avoid making whitespace changes. We can consider these sorts of changes, but this is best done in a separate task.